PyRDF/Backend/Dist.py

```diff
@@ -142,14 +134,16 @@ def __init__(self, config={}):
             'Aggregate'
         ]
 
-        # Remove the value of 'npartitions' from config dict
-        self.npartitions = config.pop('npartitions', None)
+        self.npartitions = None
```

Comment:
Why this change?

Reply:
We don't use the config dict anymore

Reply:
I can default it to `0` maybe it's better

Reply:
But we are still getting these options via the factory method for each backend, right? Then perhaps `npartitions` does not belong here anymore?